apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -118,6 +128,27 @@ const SettingsModal: React.FC<React.PropsWithChildren<InjectedModalProps>> = ({
     }
   }
 
+  const handleDiableNotifications = async () => {
+    try {
+      if (isSubscribed) await unsubscribe()
+      setAllowNotifications(false)
+      toast.toastSuccess(Events.Unsubscribed.title, Events.Unsubscribed.message?.())
+    } catch (error) {
+      const errMessage = errorBuilder(Events.UnsubscribeError, error)
```

Comment:
naming convention: use verb for functions and noun for variables
```suggestion
      const errMessage = parseErrorMessage(Events.UnsubscribeError, error)
```